test/tools/fuzzTesting/createRandomTest.cpp

```diff
@@ -218,42 +204,69 @@ void dev::test::RandomCode::parseTestWithTypes(std::string& _test, std::map<std:
 				int randomDepth = 1 + (int)dev::test::RandomCode::randomUniInt() % 10;
 				_test.replace(pos, 5, dev::test::RandomCode::rndRLPSequence(randomDepth, debug));
 				cnote << debug;
+				std::string a;
 			}
 			else
 			if (types.at(i) == "[CODE]")
-				_test.replace(pos, 6, "0x"+dev::test::RandomCode::generate(10, options));
+			{
+				int random = (int)dev::test::RandomCode::randomUniInt() % 100;
+				if (random < 90)
+					_test.replace(pos, types.at(i).length(), "0x"+dev::test::RandomCode::generate(10, options));
```

Comment:
Don't repeat `types.at(i)` 1000 times, use instead the loop `for (auto const& type: types)`